conf/zeppelin-site.xml.template

```diff
@@ -219,5 +219,11 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.websocket.max.text.message.size</name>
+  <value>1024000</value>
+  <description>Size in characters of the maximum text message to be received by websocket. Defaults to 1024000</description>
```

Comment:
btw, is this sort of the standard way to explain this? Is "character" as in byte?